### PR TITLE
Added IceCTF

### PR DESCRIPTION
**Pull request by tjgerot**
Includes IceCTF, all problems, all links to CTFtime writeups, 1 local writeup, a local and www scoreboard.

### CODE REVIEW

File: icectf-2016/crypto/alien-message-40/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# IceCTF-2016 : alien-message-40
```
Comment (dhanvi) [inline icectf-2016/crypto/alien-message-40/readme.md]: Looks like you didn't use the https://github.com/ctfs/write-ups-tools!

**Comment by dhanvi** [comment]
I would strongly suggest you the use of https://github.com/ctfs/write-ups-tools to make the work easy!

**Comment by tjgerot** [comment]
For the genctf.py what do you put for the info argument?

**Comment by abpolym** [comment]
@tjgerot The name of the information/description files - see the README.md :). If you chose to paste all the descriptions into individual files named `descriptions`, then it will be `descriptions`. I did choose `info`. Maybe I should add a `tree` output to the README :/. Btw nice job with icectf, anyways. You should, however, try to put as much change into a commit as possible - my dashboard was filled with commits :).

**Pull request merged**